extra/linux/71-liquidctl.rules

```diff
@@ -362,6 +362,9 @@ SUBSYSTEMS=="usb", ATTRS{idVendor}=="1b1c", ATTRS{idProduct}=="0c10", TAG+="uacc
 # Corsair Commander ST
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="1b1c", ATTRS{idProduct}=="0c32", TAG+="uaccess"
 
+# Corsair iCUE H100i RGB
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="1b1c", ATTRS{idProduct}=="0c35", TAG+="uaccess"
+
```

Comment:
Given the position this was added in, I think you may have missed the note saying that this rules file is generated from a script.

https://github.com/liquidctl/liquidctl/blob/4c981275b971946c7db76a34382badb124f62730/extra/linux/71-liquidctl.rules#L37-L40

_After making the other changes I suggest bellow,_ can you run that script to update this file?

Reply:
Yep, reverted to #557, i ran `~/liquidctl$ py extra/linux/generate-uaccess-udev-rules.py`

and the udev rule succesffuly appears in the output.

Reply:
Please re-run the script now and commit the changes, since its output is going to be slightly different than what's here. It's a minor thing, but we don't want that change to appear on some future, unrelated, PR.